R/stroke.R

```diff
@@ -129,19 +146,23 @@ get_linked_nodes <- function(node_id, segment_id, segments) {
 }
 
 #' @noRd
-best_link <- function(nodes, segments, links, angle_threshold = 0) {
+best_link <- function(
+  nodes, segments, links, edge_ids, angle_threshold = 0, flow_mode
```

Comment:
I didnot set `flow_mode = FALSE` in this function because the default value of `flow_mode` has already been set in `stroke()` function and hence duplicated. But I fixed the order of the arguments in `best_link` function.